setup.py

```diff
@@ -340,6 +340,7 @@ def do_setup():
             'flask-swagger==0.2.13',
             'flask-wtf>=0.14.2, <0.15',
             'funcsigs==1.0.0',
+            'graphviz>=0.12',
```

Comment:
I think it should be moved to devel - otherwise we might have similar problems like the #6009

Reply:
This will make it harder to use this command on a cloud composer. This does not install `devel` dependencies. 

After that, it seems to me that you haven't checked what this library is.  This is not graphviz that you know from the console interface (https://www.graphviz.org/).  This is not a dot program and all his friends. This is just a library that makes it easy to build dot documents. To use this change you still need to have graphviz installed on your system. However, only if you want to save the result to a file or display on the console.

Please look at setup.py: https://github.com/xflr6/graphviz/blob/master/setup.py
This is not a problematic library. It has no dependencies.

Reply:
Agreee. Merged.